kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -28,6 +28,7 @@
 namespace Kratos
 {
 	KRATOS_CREATE_LOCAL_FLAG(CalculateDiscontinuousDistanceToSkinProcessFlags, CALCULATE_ELEMENTAL_EDGE_DISTANCES, 0);
+	KRATOS_CREATE_LOCAL_FLAG(CalculateDiscontinuousDistanceToSkinProcessFlags, CALCULATE_ELEMENTAL_EXTRAPOLATED_EDGE_DISTANCES, 0);
```

Comment:
(minor) If we rename the variable I'd rename the flag accordingly.

Reply:
Actually, we could think about renaming the flag anyway.. because, yes what it directly does is lead to the calculation of the extrapolated edge distances, but what it really means is (at least right now) that the Ausas incised shape functions will be calculated and used for incised elements instead of a penalization. On the other hand, this will probably change in the future as well.

Reply:
(solved)